nilearn/regions/tests/test_region_extractor.py

```diff
@@ -285,7 +273,7 @@ def test_region_extractor_high_resolution_image():
 
 def test_region_extractor_zeros_affine_diagonal():
     n_regions = 9
-    affine = AFFINE_EYE
+    affine = np.eye(4)
```

Comment:
why not use fixture here?

Reply:
was changed in this commit

using the fixture lead to some test pollution (non independent tests) as the value of the affine is changed in the following lines

is there a better way to use fixture in this case?

make the fixture return the value and not the constant?